2020-08-21 15:40:08,942  - [TraceConfig] Init logs
2020-08-21 15:40:08,946  - [Form1_Load] DBConServer has started.
2020-08-21 15:40:24,517  - [Form1_Load] SQL is not connected.
2020-08-21 15:40:24,542  - [ExecuteDataSet] Select Distinct CollectionID from Collection
2020-08-21 15:40:24,560  - [GetDistinctCollection] A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server)
System.Data.SqlClient.SqlException (0x80131904): A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server) ---> System.ComponentModel.Win32Exception (0x80004005): The network path was not found
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.Open()
   at RESSWATCH.DataAccess.PrepareCommand(SqlCommand cmd, SqlConnection conn, SqlTransaction trans, CommandType cmdType, String cmdText, SqlParameter[] cmdParms) in C:\Users\Sami\Desktop\Shppify Management\Shppify Management\DataAccess.cs:line 161
   at RESSWATCH.DataAccess.ExecuteDataSet(CommandType commandType, String commandText, SqlParameter[] commandParameters) in C:\Users\Sami\Desktop\Shppify Management\Shppify Management\DataAccess.cs:line 138
   at BL.Collection.GetDistinctCollection() in C:\Users\Sami\Desktop\Shppify Management\Shppify Management\Product.cs:line 153
ClientConnectionId:00000000-0000-0000-0000-000000000000
Error Number:53,State:0,Class:20
